content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst

```diff
@@ -2,84 +2,89 @@
-  occur, such as a transfer to a quality control location.
-- **Shipping**: Ship products directly from stock. No intermediate steps between stock and shipping
-  occur, such as a transfer to a packing location.
-- Can only be used if not using :abbr:`FIFO (First In, First Out)`, :abbr:`LIFO (Last In, First
-  Out)`, or :abbr:`FEFO (First Expired, First Out)` removal strategies.
-- Receipts and/or deliveries are handled quickly.
-- Recommended for small warehouses with low stock levels, and for non-perishable items.
+- **Receipt**: receive products directly from vendors into stock. *No* intermediate steps between
+  receipt and input to stock occur, such as a transfer to a quality control location.
+- **Shipping**: ship products directly from warehouse stock to customer. *No* intermediate steps
+  occur before shipping, such as a transfer to a packing location.
+- Can *only* be used if **not** using First In, First Out (FIFO), Last In, First Out (LIFO), or
+  First Expired, First Out (FEFO) removal strategies.
 - Items are received or shipped directly into/from stock.
+- Suited for warehouses that  require receipts and/or deliveries to be handled quickly.
+- Recommended for small warehouses with low stock levels, and for non-perishable items.
 
 .. seealso::
    :doc:`receipts_delivery_one_step`
 
 Two-step flow
--------------
+=============
 
 The receiving and shipping rules for a two-step configuration are as follows:
 
-- **Input + stock**: Bring products to an input location *before* moving into stock. Products can be
+- **Input + stock**: bring products to an input location *before* moving into stock. Products can be
   organized by different internal storage locations, such as various shelves, freezers, and locked
   areas, before being stocked in the warehouse.
-- **Pick + ship**: Bring products to an output location before shipping. Packages can be organized
-  by different carriers or shipping docks before being shipped.
-- Minimum requirement to use lot numbers or serial numbers to track products with a
-  :abbr:`FIFO (First In, First Out)`, :abbr:`LIFO (Last In, First Out)` or
-  :abbr:`FEFO (First Expired, First Out)` removal strategy.
-- Recommended for larger warehouses with high stock levels, or when stocking large items (such as
-  mattresses, large furniture, heavy machinery, etc.).
+- **Pick + ship**: bring products to an output location before shipping. Packages can be organized
+  by different carriers or moved to shipping docks before being shipped.
+- Minimum requirement to use lot numbers or serial numbers to track products with a :abbr:`FIFO
+  (First In, First Out)`, :abbr:`LIFO (Last In, First Out)`, or :abbr:`FEFO (First Expired, First
+  Out)` removal strategy.
 - Products received will not be available for manufacturing, shipping, etc., until they are
-  transferred into stock.
+  transferred into warehouse stock.
+- Recommended for larger warehouses with high stock levels, or when stocking large items (e.g.,
```

Comment:
```suggestion
- Recommended for larger warehouses with high stock levels, or when stocking large items (e.g.
```

Reply:
Not implemented - comma is required after e.g. when there are multiple items separated by commas. Resolving comment